packages/angular_devkit/schematics/src/engine/engine.ts

```diff
@@ -85,6 +85,19 @@ export class SchematicEngine<CollectionT extends object, SchematicT extends obje
     if (!description) {
       throw new UnknownSchematicException(name, collection);
     }
+    if (!!description.extends) {
+      const parts = description.extends.split(':');
```

Comment:
Could we add a limit to this `split`? Basically, if there are multiple `:` we should allow `:` to be part of the schematics name. WDYT?